### PR TITLE
New guides for remote communication and channels

### PR DESCRIPTION
**Pull request by vpetkov**
What            | Where/Who
----------------|----------------------------------------
Reviewers       | @skanev @mitio @gmitrev

In the wake of the COVID-19 isolation, everybody is forced to work from home. Naomi will remind everyone of the communication guidelines we already have. A lot of it is scattered between pinned Slack messages and outdated Confluence pages. The aim of these new public docs is to gather a streamlined version of our remote communication style.

Adding two new pages to the guides:
* Remote communication - summary of all tips and practices from the #remote-working-tips channel in slack. The copy is refined by Mitio and approved by Naomi.
* Communication channels - a dust off refined version of the page we already have in confluence.

### CODE REVIEW

File: remote.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* Synchronous communication, like video calls and real-time chats, when all parties need to be present at the same time.
+
+* Asynchronous communication, like on Slack, Trello, Confluence, email, when everyone can participate at their own convenience. This allows collaboration across different time zones, but requires greater discipline.
+
+Long-form asynchronous communication is a better default as it brings the following benefits:
+
+- It allows for wider participation, especially across time zones and different locations.
+- It leaves a searchable written trace of what was discussed and what decisions were made.
+- It allows people to take the time to properly consider the different viewpoints, to make their research and better craft their contribution to the discussion.
+- It allows for better time and focus management, avoiding the need to be interrupted by shoulder tapping or scheduled meetings.
```
Comment (mvelikov) [inline remote.md]: The other lists use `*` for bullets. (An the style guide suggests `*` 🙊) 

```suggestion
* It allows for wider participation, especially across time zones and different locations.
* It leaves a searchable written trace of what was discussed and what decisions were made.
* It allows people to take the time to properly consider the different viewpoints, to make their research and better craft their contribution to the discussion.
* It allows for better time and focus management, avoiding the need to be interrupted by shoulder tapping or scheduled meetings.
```

**Review by mitio — APPROVED** [review]
I'll read these more carefully later. 🙈

**Review by gmitrev — APPROVED** [review]
Looks great 👏

File: remote.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Remote working
+
+Receipt Bank is a remote organization, spread across four continents and numerous time zones. This is a collection of guidelines and good practices for working remotely.
+
+## TLDR
+
+* Use long-form asynchronous communication by default. Don't schedule a meeting when a Slack message will do.
+* Over-communicate. Back-and-forth is expensive, so try to anticipate it, by including more details.
+* Switch to synchronous communication when it's more efficient. A quick zoom call sometimes will save you “hours” worth of Slack messages.
```
Comment (mitio) [inline remote.md:9]: > A quick zoom call will sometimes save you “hours” worth of Slack messages.

File: remote.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Remote usually means working from home which presents its own challenges. We have some advice that should help you.
+
+Until you get used to it, it will be pretty easy to get distracted when working from home. Start by setting some boundaries between what’s work and what’s not.
+
+Dress for work normally - working in your pajamas is a bad idea.
+
+Set up a separate place for work if possible. It could be in another room, or simply a desktop on which you’re only doing work, so your brain does not get distracted by non-work when you’re there.
+
+If you have a work computer that’s different from your personal computer, be very strict how you use it. If you have just one computer, keep a work and a personal browser separate - close the personal one during work hours and vice versa.
```
Comment (mitio) [inline remote.md:53]: > keep separate work and personal browsers

File: index.md [hunk 4]
```diff
@@ -2,4 +2,6 @@
 
 Hey there! This is a list with publicly available guides on our company culture and values.
 
+* [Remote communication](remote.md)
```
Comment (mitio) [inline index.md:5]: Have you deliberately titled this "Remote communication" as opposed to "Remote work" or "Working remotely" or something like that? The page covers a bit more than communication.

File: channels.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Communication channels
+
+This page covers what communication channels we have and how we use them.
```
Comment (mitio) [inline channels.md:3]: > covers the communication channels we have

File: channels.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+## Here's a short summary
+
+Channel | Lifespan | Type | Usage | Expectation
+--- | --- | --- | --- | ---
+Email | Temporary | Asynchronous | Announcements and notifications | Read within a day.
+Slack | Temporary | Asynchronous | General comms | Read direct messages and mentions within a few hours, keep track of channels you’re in.
+Zoom | Temporary | Synchronous | Meetings | Join on time.
+Google Docs | Temporary | Asynchronous | Collaboration | Follow up on comments in documents you’re involved in.
+GitHub | Permanent | Asynchronous | Code changes and code reviews | Read everything in your projects and all mentions within a day.
+Confluence | Permanent | Asynchronous | Processes and know-how | Different teams follow different spaces.
```
Comment (mitio) [inline channels.md:14]: Replace:

> ~~Different teams follow different spaces.~~

With:

> Read changes to pages and blog posts in the spaces you follow within a week.

File: channels.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+[Read the Slack guidelines.](slack.md)
+
+## Zoom
+
+We use Zoom for meetings. It’s a tool for video conferencing. You can start a meeting from Slack by typing `/zoom` in a channel or in a private conversation.
+
+Mute your microphone when you're not talking. It vastly improves quality for other listeners. Zoom allows you to have it muted by default so you don’t disturb the meeting when you join.
+
+Don't be late. If you can't avoid it, let other people know, so they’re not wondering where you are and when (and if) to start the meeting.
```
Comment (mitio) [inline channels.md:54]: To add to the end:

> If you can't make it at all, update your RSVP status and also let the organiser know.

File: channels.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+We use Google calendar to organize our time.
+
+Please respect other people’s calendars when inviting them to meetings. Use the calendar’s option to ‘Find a time’ to do that. It’s available in the dialog, when creating a new event.
+
+Think about the length of your meetings. Leave 5-10 minutes between events. And build in break time if you plan a longer event, like a workshop or a retrospective.
+
+There is an option in Google Calendar’s Settings called ‘Speedy meetings’. It will set the default duration to 25 or 50 minutes. Use it to avoid back-to-back meetings.
+
+Allow guests to ‘Modify event’, so people can fix the zoom link or invite others without having to chase you.
```
Comment (mitio) [inline channels.md:68]: To add to the end:

> Make sure your attendance status is up-to-date. Set it to "not attending" if you're not gonna be on the call.

**Review by mitio — COMMENTED** [review]
Nice!

A few suggestions.

File: index.md (hunk 4 above)
Comment (vpetkov) [reply index.md:5]: After so many rewrites I got lost in my own titles. Should be Remote working

File: remote.md (hunk 2 above)
Comment (vpetkov) [reply remote.md:9]: You're right here.

File: remote.md (hunk 3 above)
Comment (vpetkov) [reply remote.md:53]: I have no strong opinion. Happy to take your version after all

File: channels.md (hunk 6 above)
Comment (vpetkov) [reply channels.md:14]: 👍

File: channels.md (hunk 8 above)
Comment (vpetkov) [reply channels.md:68]: I've updated all your other remarks with ninja commits. I'm not sure about the wording on this one. And it's getting a little bit duplicated with the line above. Let me think about another version

**Pull request merged**